src/Resources/views/Block/timeline.html.twig

```diff
@@ -28,55 +28,62 @@ file that was distributed with this source code.
         {% endif %}
 
         <div class="panel-body">
+            <ul class="timeline">
+                {% for entry in entries %}
+                    {% if currentDay is not defined or currentDay != entry.createdAt|format_date %}
+                        {% set currentDay = entry.createdAt|format_date %}
+                        <li class="time-label">
+                            <span class="bg-red">{{ currentDay }}</span>
+                        </li>
+                    {% endif %}
 
-            {% sonata_template_box 'This is the timeline block.' %}
-
-            <div class="row">
-                <div class="col-md-12">
-                    <ul class="timeline">
-                        {% for entry in entries %}
-
-                            {% if currentDay is not defined or currentDay != entry.createdAt|format_date %}
-                                {% set currentDay = entry.createdAt|format_date %}
-                                <li class="time-label">
-                                    <span class="bg-red">{{ currentDay }}</span>
-                                </li>
-                            {% endif %}
-
-                            {% set subject = entry.getComponent('subject') %}
-                            {% set target = entry.getComponent('target') %}
-                            {% set target_text_component = entry.getComponent('target_text') %}
+                    {% set subject = entry.getComponent('subject') %}
+                    {% set target = entry.getComponent('target') %}
+                    {% set target_text_component = entry.getComponent('target_text') %}
 
-                            {% if subject.data %}
-                                {% set subject_text = sonata_timeline_generate_link(subject, entry) %}
-                            {% else %}
-                                {% set subject_text = '<abbr title="' ~ 'element_reference_deleted'|trans({'%reference%': target.hash}, "SonataTimelineBundle") ~ '">' ~ 'element_deleted'|trans({}, "SonataTimelineBundle") ~ '</abbr>' %}
-                            {% endif %}
+                    {% if subject.data %}
+                        {% set subject_text = sonata_timeline_generate_link(subject, entry) %}
+                    {% else %}
+                        {% set subject_text = '<abbr title="' ~
+                            'element_reference_deleted'|trans({'%reference%': target.hash}, "SonataTimelineBundle") ~ '">' ~
+                            'element_deleted'|trans({}, "SonataTimelineBundle") ~
+                            '</abbr>'
+                        %}
```

Comment:
For bonus points, if you want to, you could replace this with a set block: `{% set subject_text %}the syntactically colorized html{% set %}`